src/KingSystem/ActorSystem/actInfoData.h

```diff
@@ -74,6 +74,10 @@ class InfoData {
         sead::SafeArray<sead::SafeString, 16> times;
         s32 num_weathers;
         sead::SafeArray<sead::SafeString, 16> weathers;
+
+        bool containsCurrentTimeOrWeather(const sead::Vector3f& pos);
```

Comment:
All three functions can also be marked as const since they don't modify object state. Apologies for missing this earlier.